installer/conf/logrotate.conf

```diff
@@ -6,3 +6,12 @@
     size 100M
     copytruncate
 }
+
+/var/opt/microsoft/omsagent/log/update_run_progress.log {
```

Comment:
I have confirmed with Brandon that this work is being deffered and is not necessary. Multi-homing does not have to be fully supported with this PR.